app/javascript/mastodon/reducers/contexts.ts

```diff
@@ -129,12 +166,30 @@ const updateContext = (state: Draft<State>, status: ApiStatusJSON): void => {
 export const contextsReducer = createReducer(initialState, (builder) => {
   builder
     .addCase(fetchContext.fulfilled, (state, action) => {
-      normalizeContext(state, action.meta.arg.statusId, action.payload.context);
+      if (action.payload.prefetchOnly) {
```

Comment:
This flag changes the behaviour of the entire action – I'm not sure if it's better in such cases to create a separate action instead, however in that case the action definition (the `createDataLoadingThunk` call) would be duplicated.